libs/gl-plugin/src/node/mod.rs

```diff
@@ -475,10 +464,16 @@ impl Node for PluginNodeServer {
     ) -> Result<Response<pb::Empty>, Status> {
         self.limit().await;
         let gl_config = req.into_inner();
-        let rpc = self.get_rpc().await;
+        let mut rpc = self.get_rpc().await.map_err(|e| {
+            tonic::Status::new(
+                tonic::Code::Internal,
```

Comment:
And this is the error messages all "early calls" will run into. Not the best UX.